Makefile

```diff
@@ -24,9 +24,12 @@ config.yml:
 install: .venv/bin/python .venv/bin/pip-licenses .venv/bin/elastic-ingest
 	.venv/bin/pip-licenses --format=plain-vertical --with-license-file --no-license-path > NOTICE.txt
 
+.venv/bin/hatch: .venv/bin/python
+	.venv/bin/pip install -r requirements/$(ARCH).txt
+
 .venv/bin/elastic-ingest: .venv/bin/python
 	.venv/bin/pip install -r requirements/$(ARCH).txt
-	.venv/bin/python setup.py develop
+	.venv/bin/pip install --editable .
```

Comment:
Why is the `--editable` ? 

`make run` uses this command to run the app in prod use cases, in such scenario we probably don't want to include `--editable` flag?